packages/build/config/tslint.build.json

```diff
@@ -15,9 +15,8 @@
     // User-land promises like Bluebird implement "PromiseLike" (not "Promise")
     // interface only. The string "PromiseLike" bellow is needed to
     // tell tslint that it's ok to `await` such promises.
-    "await-promise": [true, "PromiseLike"],
-    "no-floating-promises": true,
-    "no-unused-variable": true,
```

Comment:
I had looked into making this PR but decided against it based on https://github.com/palantir/tslint/issues/4100 ... TypeScript doesn't recommend using the `noUnusedLocals` flag as an alternative to linting which provides finer line-by-line control. Maybe we should keep this while also using the `noUnusedLocals` flag ... at least until there is a final decision from `tslint` on deprecating or undeprecating the rule.